svc-bip-api/src/test/java/gov/va/vro/bip/config/RabbitMqApiConfigTest.java

```diff
@@ -17,7 +17,8 @@ class RabbitMqApiConfigTest {
   @Test
   @SuppressWarnings({"unchecked", "rawtypes"})
   public void testRabbitMqConfig() {
-    RabbitMqConfig rmqConfig = new RabbitMqConfig();
+    RabbitMqConfig rmqConfig =
```

Comment:
Do we need a test to verify the payload invalid rejection?

Reply:
In my research it doesn't seem like there is a direct way to test an exceptionHandler on the rabbitmq listeners except by including a proxy listener as defined here https://github.com/garyrussell/spring-amqp/commit/ced13241e7c3e32c140752ec601c33774b8b77c0.

Reply:
If I understand correctly the listener is returning an exception thrown by rabbitMq due to missing a reply-to field.